keras_cv/layers/preprocessing/grayscale.py

```diff
@@ -14,13 +14,13 @@
 
 import tensorflow as tf
 
-from keras_cv.layers.preprocessing.base_image_augmentation_layer import (
-    BaseImageAugmentationLayer,
+from keras_cv.layers.preprocessing.vectorized_base_image_augmentation_layer import (
+    VectorizedBaseImageAugmentationLayer,
 )
 
 
 @tf.keras.utils.register_keras_serializable(package="keras_cv")
-class Grayscale(BaseImageAugmentationLayer):
+class Grayscale(VectorizedBaseImageAugmentationLayer):
```

Comment:
I suppose it's good that this layer's tests don't need to be changed.

Can you please verify the performance of this new solution by A/B testing the old+new grayscale layer?

Reply:
Done